docker/docker_client.go

```diff
@@ -48,6 +48,15 @@ const (
 	extensionSignatureTypeAtomic    = "atomic" // extensionSignature.Type
 )
 
+// systemPerHostCertDirPath is the path to certs.d, used for storing per-host TLS keys and certificates.
+// You can override this at build time with
+// -ldflags '-X github.com/containers/image/docker.systemPerHostCertDirPath=$your_path'
+var systemPerHostCertDirPath = builtinPerHostCertDirPath
+
+// builtinPerHostCertDirPath is the path to certs.d.
+// DO NOT change this, instead see systemPerHostCertDirPath above.
+const builtinPerHostCertDirPath = "/etc/docker/certs.d"
```

Comment:
I can't imagine cases where people want to change this. If people have certs in another location they'll use `DockerCertPath` and I won't leave them the ability to ovveride this at build time - they'll come here with some issue and we first need to realize they built skopeo with another buildinPerHostCertDirPath and work from there. I'd rather have them explicitely use `DockerCertPath` instead.